daml-script/daml/Daml/Script.daml

```diff
@@ -184,7 +202,10 @@ data PartyDetails = PartyDetails
     isLocal : Bool -- ^ True if party is hosted by the backing participant.
   deriving (Eq, Ord, Show)
 
--- | In wallclock mode, this is UTC time. In static time mode, this is the UNIX epoch.
+-- | In wallclock mode, this is UTC time. In static time mode, this will
+-- query the ledger time service for the current time when running over
+-- gRPC. When running over the JSON API, it will always
+-- return the Unix epoch.
```

Comment:
When running in wallclock mode over the JSON API, do you get UTC time or Unix epoch? To me, that's not really clear from the comment.

Reply:
#6425